src/ALB.Api/UseCases/Endpoints/Admin/ApplicationUser/CreateApplicationUser/CreateApplicationUserEndpoint.cs

```diff
@@ -0,0 +1,20 @@
+using FastEndpoints; 
+
+namespace ALB.Api.UseCases.Endpoints.Admin.ApplicationUser.CreateApplicationUser;
+
+public class CreateApplicationUserEndpoint : Endpoint<CreateApplicationUserRequest, CreateApplicationUserResponse>
+{
+    public override void Configure()
+    {
+        Post("/admin/application-user/create-application-user");
```

Comment:
Ich würde die Pfade etwas einfacher halten, entsprechend gängiger REST-Konventionen.

Hier ein auszug mit beispielen

![image](https://github.com/user-attachments/assets/2d0d07dc-1a96-475e-a0a5-a7ee9ab01150)


```suggestion
        Post("api/users");
```